recipes/libpng/all/conanfile.py

```diff
@@ -92,15 +97,21 @@ def source(self):
     def _patch(self):
         for patch in self.conan_data.get("patches", {}).get(self.version, []):
             tools.patch(**patch)
-        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
-                              "find_library(M_LIBRARY m)",
-                              "set(M_LIBRARY m)")
+        if self.version != "1.5.2":
```

Comment:
```suggestion
        if tools.Version(self.version) > "1.5.2":
```
Because when bumping version only, people will not see conanfile.py